### PR TITLE
allow avatar image download

### PR DESCRIPTION
**Pull request by ssoonmi**

### CODE REVIEW

**Comment by anjosh** [comment]
Thanks @ssoonmi! I'll review this soon

**Comment by anjosh** [comment]
I had to make changes to personAvatar.vue to fix the image related performance bug I mentioned to you before. So could you rebase this off of master and update this PR?
I think you'll have to think of a slightly different approach than what you have now since we're not returning the full image on the `/guests` call anymore.

**Comment by anjosh** [comment]
@ssoonmi tagging you in case you don't get notified of my above comment.

**Comment by ssoonmi** [comment]
Rebased and refactored

File: components/personAvatar.vue [hunk 1]
```diff
@@ -1,12 +1,73 @@
...
   export default {
-    props: ['person']
+    props: ['person'],
+    data () {
+      const { avatar, avatarURL } = this.person
+      return {
+        image: avatar || avatarURL
+      }
+    },
+    watch: {
```
Comment (anjosh) [inline components/personAvatar.vue]: @ssoonmi what's the purpose of the watch here?

Comment (ssoonmi) [reply components/personAvatar.vue]: the person changes in the component when you open a contact, then open another person's contact. `this.image` needs to be updated when person changes.

**Comment by anjosh** [comment]
Looks great!

**Pull request merged**